daktari/checks/terraform.py

```diff
@@ -7,26 +7,40 @@
 from daktari.os import OS
 
 
+class TfenvInstalled(Check):
+    name = "tfenv.installed"
+
+    suggestions = {
+        OS.OS_X: "<cmd>brew install tfenv</cmd>",
+        OS.UBUNTU: """<cmd>git clone https://github.com/tfutils/tfenv.git ~/.tfenv</cmd>
+                      <cmd>ln -s ~/.tfenv/bin/* /usr/local/bin</cmd>""",
```

Comment:
Won't you need `sudo` for that?

Reply:
ah yeah maybe, can't hurt to add it :+1: